dts/vendor/nordic/nrf54lm20a_partition.dtsi

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&cpuapp_rram {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* nRF54LM20A has 2036 kB of non-volatile memory (RRAM) but the last
+		 * 96 kB are reserved for the FLPR MCU, so we have ~1940 kB available.
+		 */
```

Comment:
as above, no

Reply:
total size of partitions below: 1696KiB, according to comment above, there is 1940 available, so where has the other 244 magically vanished off to? The original partitioning came to a total size of 1896 so something is very wrong here